lib/pancake.js

```diff
@@ -255,27 +255,28 @@ class Pancake {
       // if metadata is not defined, skip this build
       return;
     }
-    const properties = {
-      'createdAt': {
-        'type': 'Date',
-        'defaultFn': 'now',
-      },
-      'modifiedAt': {
-        'type': 'Date',
-        'defaultFn': 'now',
-      },
-    };
+    const properties = {};
     const indexes = {};
     const security = {
       _: '',  // for all fields
     };
 
     let plural, baseModel;
     if (!isEmbeddable) {
-      if (metadata.base === 'PersistedModel') {
-        plural = Utils.pluralize(metadata.name);
-      } else {
+      switch (metadata.base) {
+      case 'AccessToken':
+      case 'Application':
+      case 'Role':
+      case 'RoleMapping':
+      case 'Scope':
+      case 'User':
+      case 'UserCredential':
+      case 'UserIdentity':
```

Comment:
I would suggest we should define the list in a JSON file just like https://github.com/weflex/pancake/blob/master/lib/natives.json, and we can update the only file when loopback updates it.

The recommend name of this file in my opinion is "models.loopback.json".